app/models/post.rb

```diff
@@ -1,2 +1,14 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    author.posts_counter = author.posts.count
+    author.save
+  end
```

Comment:
- [ ] You have correctly added the `update_posts_counter` method as required. However, a more efficient way to achieve this is using the `after_save` callback. Additionally, the post counter should be updated using the increment method, and it's a good practice to make this method private.  :blush: